android/settings.gradle

```diff
@@ -1,6 +1,7 @@
 include ':app-example'
 include ':templates:snapshot'
 include ':templates:release-0.8'
+include ':templates:no-dagger'
```

Comment:
I wouldn't make it a separate module. I'd make it a template choice in the plugin dropdown, otherwise versioning will be messed up.

Reply:
Makes sense